test/system/account_management_test.rb

```diff
@@ -1,53 +1,46 @@
-      resize_for(display_details)
+  device_test "user can edit their account and change account settings during registration" do
+    be_invited_to_sign_up
 
-      be_invited_to_sign_up
+    visit root_path
+    sign_up_from_homepage
 
-      visit root_path
-      sign_up_from_homepage_for(display_details)
+    fill_in "Email", with: "andrew.culver@gmail.com"
+    fill_in "Set Password", with: example_password
+    fill_in "Confirm Password", with: example_password
+    click_on "Sign Up"
 
-      fill_in "Email", with: "andrew.culver@gmail.com"
-      fill_in "Set Password", with: example_password
-      fill_in "Confirm Password", with: example_password
-      click_on "Sign Up"
-
-      if billing_enabled?
-        complete_pricing_page
-      end
-
-      # we should now be on an onboarding step.
-      assert page.has_content?("Tell us about you")
-      fill_in "First Name", with: "Testy"
-      fill_in "Last Name", with: "McTesterson"
-      fill_in "Your Team Name", with: "The Testing Team"
-      page.select "Brisbane", from: "Your Time Zone"
-      click_on "Next"
-
-      assert page.has_content?("The Testing Team’s Dashboard")
-
-      user = User.find_by(email: "andrew.culver@gmail.com")
+    if billing_enabled?
+      complete_pricing_page
+    end
 
-      visit edit_account_user_path(user)
+    # we should now be on an onboarding step.
+    assert_text "Tell us about you"
+    fill_in "First Name", with: "Testy"
+    fill_in "Last Name", with: "McTesterson"
+    fill_in "Your Team Name", with: "The Testing Team"
+    page.select "Brisbane", from: "Your Time Zone"
+    click_on "Next"
 
-      fill_in "Email", with: "andrew.culver.new@gmail.com"
-      fill_in "First Name", with: "Testy.new"
-      fill_in "Last Name", with: "McTesterson.new"
-      page.select "Tokyo", from: "Your Time Zone"
+    assert_text "The Testing Team’s Dashboard"
 
-      click_on "Update Profile"
+    user = User.find_by(email: "andrew.culver@gmail.com")
 
-      assert page.has_content? "User was successfully updated."
+    visit edit_account_user_path(user)
+    fill_in "Email", with: "andrew.culver.new@gmail.com"
+    fill_in "First Name", with: "Testy.new"
+    fill_in "Last Name", with: "McTesterson.new"
+    page.select "Tokyo", from: "Your Time Zone"
+    click_on "Update Profile"
 
-      visit edit_account_user_path(user)
+    assert_text "User was successfully updated."
+    visit edit_account_user_path(user)
 
-      assert page.find("#user_email").value == "andrew.culver.new@gmail.com"
-      assert page.find("#user_first_name").value == "Testy.new"
-      assert page.find("#user_last_name").value == "McTesterson.new"
-      assert page.find("#user_time_zone").value == "Tokyo"
-    end
+    assert_field "Email", with: "andrew.culver.new@gmail.com"
+    assert_field "First Name", with: "Testy.new"
+    assert_field "Last Name", with: "McTesterson.new"
+    assert_field "Your Time Zone", with: "Tokyo"
```

Comment:
Same deal as the `assert_text` change, this now pretty much becomes a 1-1 of the `fill_in`/`select` update above.